### DISCUSSION TITLE
improving closed sidebar UI

### DISCUSSION THREAD

**Discussion by yash-dani** [jellypbc/poster#258] [General]
is it possible to add something to the sidebar when it is closed to make it clear its the sidebar? right now just having it is just a triangle may be confusing, but the sidebar overall seems really really useful though!
![image](https://user-images.githubusercontent.com/41090933/88864169-93ba0300-d1d2-11ea-97dd-a24d259158e3.png)

**Comment by cindywu** [comment]
@dml-1 discussed in person adding text 'open sidebar' or 'close sidebar' when the user hovers on the caret. exact wording for the action is tbd.

**Comment by cindywu** [comment]
I made some style improvements to sidebar in https://github.com/jellypbc/poster/pull/296 and https://github.com/jellypbc/poster/pull/304. 

This is still an open issue: https://github.com/jellypbc/poster/issues/262

**Comment by cindywu** [comment]
improvements for sidebar ui are coming! https://www.figma.com/file/gt3p3JDh5W5i5N2wXxE7I5/Design-System?node-id=235%3A194

https://github.com/jellypbc/poster/pull/364
